set.mm

```diff
@@ -22680,23 +22693,37 @@ of the unique existential quantifier (note that ` y ` and ` z ` need not
        using the at-most-one quantifier.  (Contributed by NM, 12-Aug-1993.)
        This used to be the definition of the unique existential quantifier,
        while ~ df-eu was then proved as ~ dfeu .  (Revised by BJ, 30-Sep-2022.)
-       (Proof shortened by Wolf Lammen, 3-Jan-2023.) $)
+       (Proof shortened by Wolf Lammen, 3-Jan-2023.)  Remove use of ~ ax-11 .
```

Comment:
Wasn't exsb provable without ax-11 as well? See #3485 .

Reply:
Hmm, I didn't prove that exsb didn't need ax-11, but rather that exsb and dfmolem are similar (but dfmolem doesn't need ax-11)

Reply:
Ok then.  I remembered it differently